.travis.yml

```diff
@@ -8,6 +8,8 @@ cache:
 services:
   - docker
 # UI dependencies
+env:
+  - IMAGE_PREFIX=quay.io/helmpack
```

Comment:
Why do you need IMAGE_PREFIX to be a env variable? Does it change between runs?

Reply:
because we'll later reuse it for chartsvc and chart-repo